README.md

```diff
@@ -1,100 +1,137 @@
 
-This repository represents ***the OSTIS Technology web-oriented platform*** implementation. You can use it to store, represent 
-and process your information in formal language of universal representation of knowledge. The OSTIS Technology platform 
-is associated with the IMS system. 
+OSTIS Web platform contains:
 
-The OSTIS Technology platform contains:
-1) [Knowledge base with top-level ontologies](https://github.com/ostis-ai/ims.ostis.kb) to specify any information models 
-in scope of any subject domains;
-2) [Knowledge processing machine](https://github.com/ostis-ai/sc-machine) with semantic network storage and their 
-processing with tools to interact with it;
-3) [Web-oriented semantic interface](https://github.com/ostis-ai/sc-web) to users interact with the IMS system.
+1) [Knowledge base with top-level ontologies](https://github.com/ostis-ai/ims.ostis.kb) to help you develop a broad variety of information models
+2) [Knowledge processing machine](https://github.com/ostis-ai/sc-machine) with semantic network storage and agent-based knowledge processing
+3) [Web-oriented semantic interface](https://github.com/ostis-ai/sc-web) for users to interact with the intelligent system.
 
-# Installation
-## Using Docker
-You will need [Docker](https://docs.docker.com/get-docker/) installed and running on your machine.
+To learn more about the platform, check out our [documentation](https://raw.githubusercontent.com/ostis-ai/ostis-web-platform/develop/docs/main.pdf).
 
-```sh
-git clone https://github.com/ostis-ai/ostis-web-platform
-cd ostis-web-platform
-docker compose pull
-```
+## Installation
 
-<details> 
-<summary> Building images locally </summary>
++ Quick start using Docker Compose
 
-e.g. when you want to launch a custom branch of sc-machine/sc-web
-#### Requirements
-1. **If you're using Windows**, please make sure you are using UNIX line endings inside the repository and `longpaths` are enabled:
+   Requirements: you will need [Docker](https://docs.docker.com/get-docker/) installed and running on your machine.
+
+   ```sh
+   git clone https://github.com/ostis-ai/ostis-web-platform
+   cd ostis-web-platform
+   # download images from Docker Hub
+   docker compose pull
+   # build knowledge base
+   docker compose run machine build
+   # launch web platform stack
+   docker compose up
    ```
-   git config --local core.autocrlf true
-   git config --local core.longpaths true
-   git add --normalize .
+
+   <details>
+   <summary> Building docker images locally </summary>
+
+   This may come in handy e.g. when you want to use a custom branch of sc-machine or sc-web.
+
+   ### Requirements
+
+   1. **If you're using Windows**, please make sure you are using UNIX line endings inside the repository and `longpaths` are enabled:
+
+      ```
+      git config --local core.autocrlf true
+      git config --local core.longpaths true
+      git add --normalize .
+      ```
```

Comment:
As I know, we should face these problems on Windows even without building (e.g. cloning `ims.ostis.kb`). So maybe it'd be better not to hide these lines in `details` section.

Reply:
This is definitely a valid concern, but we should probably test it. Also I've seen a weird behaviour on a Windows machine - it required the `git add` flag to be `--renormalize`, not `--normalize`. Long story short - testing required before merging this, and, well, I don't have a Windows machine. It'd be great to get some help with that 😄

Reply:
Soo... What are we doing about it? Did we test that **not** having these settings actually breaks the KB building process on Windows?